docker/ml-testing-toolkit/test-cases/collections/provisioning/hub_setup.json

```diff
@@ -576,97 +576,7 @@
           "path": "/oracles",
           "method": "post",
           "body": {
-            "oracleIdType": "MSISDN",
```

Comment:
I guess we really don't need multiple oracles if we are only testing with one.
You're removing to keep it in line since there was some GP id conflicts correct?
Bit iffy on this but I'm fine with it.

Update the description on line 567/569 if you're changing to `ACCOUNT_ID`

Reply:
The conflict is with GP, which is also testing with MSISDN oracle. Once the perf oracle is provisioned, the GP tries to use it instead of the in-cluster oracle, which causes some of the tests to fail. I am not sure if it fails only when the perf simulators are down, but I found a way to avoid this conflict by using a different type.

I updated the description per your advice.